src/renderer/sw_engine/tvgSwShape.cpp

```diff
@@ -255,49 +334,22 @@ static SwOutline* _genDashOutline(const RenderShape* rshape, const Matrix* trans
     //No actual shape data
     if (cmdCnt == 0 || ptsCnt == 0) return nullptr;
 
+    auto startPts = pts;
+    auto startCmds = cmds;
+
     SwDashStroke dash;
     auto offset = 0.0f;
-    auto trimmed = false;
+    auto simultaneous = !rshape->stroke->trim.individual;
```

Comment:
Sorry, failed to rebase by merge-conflict. Could you please update the patch again?